pyro/contrib/forecast/forecaster.py

```diff
@@ -11,7 +11,7 @@
 import pyro.distributions as dist
 import pyro.poutine as poutine
 from pyro.infer import SVI, Trace_ELBO
-from pyro.infer.autoguide import AutoNormal, init_to_sample
+from pyro.infer.autoguide import AutoDiagonalNormal, init_to_sample
```

Comment:
This works around a bug in `AutoNormal` triggered by the LKJ distribution. I'll fix that bug later...